### ISSUE TITLE
[PayPal] Attach additional PayPal info within placing an order endpoint

### ISSUE THREAD

**Issue by mkucmus** [DivanteLtd/shopware-pwa#629]
In order to finalize the payment on SW6 side - the API should treat that kind of request as paypal based. It can be done via attaching some data, that SW6 understands and can parse in the payment listeners.

```
isPayPalSpbCheckout: 1
paypalPaymentId: PAYID-L2OL3CA5YG011649J251950W
paypalPayerId: QCLJTFXNNTV48
```
is a sample request payload. We can follow this logic, as it should be working out of the box also within the Sales-Channel-API (not only the storefront - to FIND OUT)

Acceptance criteria:
- [ ] isPayPalSpbCheckout, paypalPaymentId, paypalPayerId - are attached to the placing order's request as another parameters (in the POST payload)
- [ ] if there is a redirection as a response given - it's probably because the extra finalize-transaction endpoint is needed to be called as well - it sets the order as a paid one. - FIND OUT whether it's not necessary for sales-channel-api. 

is blocked by #628

**Comment by mkucmus** [comment]
won't be necessary if assumptions from #628 are agreed

**Comment by mkucmus** [comment]
smart buttons are no longer needed due to new assumptions from #679
